builders/xcm/fees.md

```diff
@@ -142,33 +150,35 @@ On Kusama, the benchmarked base weights are broken up into two categories: fungi
 
 With the instruction weight cost established, you can calculate the cost of the instruction in KSM. 
 
-In Kusama, the [`ExtrinsicBaseWeight`](https://github.com/paritytech/polkadot/blob/master/runtime/kusama/constants/src/weights/extrinsic_weights.rs#L56){target=_blank} is set to `{{ networks.kusama.extrinsic_base_weight.display }}` which is mapped to 1/10th of a cent. Where 1 cent is `10^12 / 30,000`. Therefore, a constant exists in the formula for calculating the final fee in DOT:
+In Kusama, the [`ExtrinsicBaseWeight`](https://github.com/paritytech/polkadot/blob/master/runtime/kusama/constants/src/weights/extrinsic_weights.rs#L56){target=_blank} is set to `{{ networks.kusama.extrinsic_base_weight.display }}` which is [mapped to 1/10th](https://github.com/paritytech/kusama/blob/master/runtime/polkadot/constants/src/lib.rs#L85){targer=blank} of a cent. Where 1 cent is `10^12 / 30,000`.
+
+Therefore, to calculate the cost of executing an XCM instruction, you can use the following formula:
 
 ```
-Planck-KSM-Weight =  PlanckKSM-Mapped * (1 / ExtrinsicBaseWeight)
+XCM-KSM-COST = XCMInstrWeight * KSMWeightToFeeCoefficient
 ```
 
-Where `Planck-KSM-Weight` is calculated as follows:
+Where `KSMWeightToFeeCoefficient` is a constant (map to 1 cent), and can be calculated as:
 
 ```
-Planck-KSM-Weight = (10^12 / 30000 * 10) * (1 / {{ networks.kusama.extrinsic_base_weight.numbers_only }})
+KSMWeightToFeeCoefficient = ( 10^12 / ( 10 * 30000 )) * ( 1 / KSMExtrinsicBaseWeight )
```

Comment:
I would do the same thing as you did in the DOT section here with the "Using the actual values:" and then showing the actual values

Reply:
nice catch!